@ecomplus/storefront-template/template/pages/@/layout/footer.ejs

```diff
@@ -180,6 +181,11 @@ const contacts = _.cms('contacts') || {}
       <span>
         <%= footer.credits || `© ${_.store.name} / ${contacts.address} / ${contacts.doc_number}` %>
       </span>
+      <% if (themeCustom==='ecom-beauty' ) { %>
+        <div class="ecom-credits-container">
+          <%- await include('@/layout/inc/ecom-credits') %>
```

Comment:
Aqui ainda tá diferente a cor do logo da ecom, por conta do src do img
Nós temos uma opção de logo branco pra colocar nessa parte?

Reply:
A mesma situação se aplica aos payment methods do footer

Reply:
Você pode baixar os assets do próprio XD na verdade e inserí-los na PR em https://github.com/ecomplus/storefront/tree/master/%40ecomplus/storefront-template/template/public/assets , eu publico eles em um CDN depois se for o caso